command.c

```diff
@@ -643,6 +645,45 @@ bool command_show_osd_msg(command_t *cmd, const char* arg)
     return true;
 }
 
+bool command_load_state_slot(command_t *cmd, const char *arg)
+{
+   retro_ctx_size_info_t info;
+   char reply[128]              = "";
+   runloop_state_t *runloop_st  = runloop_state_get_ptr();
+   const rarch_system_info_t
+      *system                   = &runloop_st->system;
+   unsigned int slot            = (unsigned int)strtoul(arg, NULL, 10);
+   char *reply_at               = reply + snprintf(reply, sizeof(reply) - 1, "LOAD_STATE_SLOT %d", slot);
+   bool savestates_enabled      = core_info_current_supports_savestate();
+   bool ret                     = false;
+   char state_path[16384];
+   state_path[0]                = '\0';
+   if (savestates_enabled)
+   {
+      runloop_get_current_savestate_path(state_path,
+            sizeof(state_path));
+
+      core_serialize_size(&info);
+      savestates_enabled = (info.size > 0);
+   }
+   if (savestates_enabled)
+   {
+      ret = content_load_state(state_path, false, false);
+      if(ret)
+      {
+          command_post_state_loaded();
+      }
+   }
+   else
+   {
```

Comment:
Same here, get rid of one line code brackets.

Reply:
Sure thing.